sansa-ml-spark/src/main/scala/net/sansa_stack/ml/spark/clustering/utils/dataProcessing.scala

```diff
@@ -0,0 +1,178 @@
+package net.sansa_stack.ml.spark.clustering.utils
+
+import java.io.{File, FilenameFilter}
+
+import com.typesafe.config.Config
+import org.apache.jena.graph.Triple
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.SparkSession
+
+import net.sansa_stack.ml.spark.clustering.datatypes.{Categories, CoordinatePOI, Poi}
+import net.sansa_stack.rdf.spark.io.NTripleReader
+
+
+
+/**
+  * load TomTom dataset
+  * @param spark SparkSession
+  * @param conf Configuration
+  */
+class dataProcessing(val spark: SparkSession, val conf: Config) extends Serializable {
```

Comment:
I think we should folllow our set scala style guidelines on naming convention as well :) Classes should be named in upper camel case: https://docs.scala-lang.org/style/naming-conventions.html#classestraits .
I will take care of that :)